lib/plugins/entities.js

```diff
@@ -199,6 +209,7 @@ function inject (bot, { version }) {
   bot._client.on('spawn_entity_experience_orb', (packet) => {
     const entity = fetchEntity(packet.entityId)
     entity.type = 'orb'
+    entity.name = 'experience_orb'
```

Comment:
hardcoded it because it's not consistent per version and we don't have the id here